admin_manual/release_notes.rst

```diff
@@ -17,6 +18,110 @@ Release Notes
+
+``<?php print_unescaped($this->inc('html.mail.footer', ['app' => 'core'])); ?>``
+
+- Add the following at the end of each plain text template to add the footer 
+
+``<?php print_unescaped($this->inc('plain.mail.footer', ['app' => 'core'])); ?>``
+
+- In a custom theme change ``getShortFooter`` and ``getLongFooter`` in *defaults.php* `without links <https://github.com/owncloud/theme-example/blob/master/defaults.php#L124>`_  to `include the links <https://github.com/owncloud/core/blob/master/lib/private/legacy/defaults.php#L256>`_
+
+Changed behavior of "Exclude groups from sharing" option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The option "Exclude groups from sharing" in the administration settings 'Sharing' section enables administrators to exclude groups of users from the ability to initiate file shares. In previous versions this restriction only applied to users that are members of exactly these groups (membership of one or more non-excluded groups bypassed the restriction). This behavior has been changed to be more restrictive and to better cover the expectations of administrators. With ownCloud Server 10.0.9 it will apply to all users that are member of at least one of the excluded groups.
+
+Changes to the sharing autocomplete mechanism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With ownCloud Server 10.0.8 the `value for minimum characters to trigger the sharing autocomplete mechanism has been made configurable and set to “4” by default <https://doc.owncloud.com/server/latest/admin_manual/release_notes.html#new-config-setting-to-specify-minimum-characters-for-sharing-autocomplete>`_. As this security-enhancing change came at the expense of usability and might only be required in special scenarios the default value has been set back to "2". For higher security requirements the *config.php* option ``'user.search_min_length' => 2,`` can be adapted. To further improve usability a hint has been added to inform users about the required character count to get suggestions.
+
+Improvements for *occ user:list*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To improve the usability of ``occ user:list`` the output has been made configurable by using the ``-a`` option to include certain attributes. This change has mainly been introduced to facilitate automation tasks. Check the ``--help`` option for more information.
+
+Additional events for audit logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New events have been added to be used for audit logging, among others. These include *settings changes by users*, *sending public links via mail* as well as *accepting/rejecting shares*. When logs are forwarded to external analyzers like Splunk, administrators can check to add the new events. The latest version of the Auditing extension (admin_audit) is required.
+
+Theming improvements and changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- HTML templates for *lost password* mails have been added
```

Comment:
does this need an introductory paragraph telling that the following information is important in case a custom theme is used and might need manual adjustments ?

Reply:
Sounds fair.